EIPS/eip-4799.md

```diff
@@ -0,0 +1,241 @@
+---
+eip: 4799
+title: Non-Fungible Token Wrapping Standard
+description: A standardized interface for contracts that own NFTs and mints versions of them as wrapped NFTs
+author: David Buckman (@davidbuckman), Isaac Buckman (@isaacbuckman)
+discussions-to: https://ethereum-magicians.org/t/erc-4799-non-fungible-token-wrapping-standard/8396
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-02-13
+requires: 165
+---
+
+## Abstract
+
+The following defines a standard interface for NFT wrapping contracts - contracts with which users can escrow their NFTs and receive a new provably authentic wrapped NFT that represents their ownership. This allows for the introduction of additional functionality for any preexisting NFT, without sacrificing authenticity.
+
+The standard works by exposing an interface that allows client applications to track a chain of authenticity from the original NFT contract to the final owner. The owner can transfer their NFT to an ERCX contract, which then assigns ownership according to its own code - say, back to the original owner. Then, applications that want to know who the owner of an NFT is will ask the original contract, and upon being told that the owner is an ERCX contract, will ask that ERCX contract who the owner of that token is - in our example, still the original owner. In this way, the original owner will still receive all the utility associated with their NFT, there is still only one owner for a given NFT, but the owner can now enjoy any added functionality provided by the implementation of the ERCX contract (say, the ability to rent ownership of their NFT).
+
+The standard works by exposing an interface that allows client applications to track a chain of authenticity from the original NFT contract to the final owner. Here is an example of a simple rental contract. The owner, Owner A can transfer their NFT to an ERCX contract, which then assigns ownership according to its own code - say, back to the Owner B. Then, applications that want to know who the owner of an NFT is will ask the original contract, and upon being told that the owner is an ERCX contract, will ask that ERCX contract who the owner of that token is - in our example, still the original owner. In this way, the original owner will still receive all the utility associated with their NFT, there is still only one owner for a given NFT, but the owner can now enjoy any added functionality provided by the implementation of the ERCX contract (say, the ability to rent ownership of their NFT).
```

Comment:
I'd try to collapse this down a bit. The abstract should be a terse technical summary describing what the EIP specifies.

Reply:
Cut this down to a single paragraph